src/lib/collision_prevention/CollisionPrevention.cpp

```diff
@@ -400,11 +400,17 @@ CollisionPrevention::_addDistanceSensorData(distance_sensor_s &distance_sensor,
 		int lower_bound = (int)round((sensor_yaw_body_deg  - math::degrees(distance_sensor.h_fov / 2.0f)) / BIN_SIZE);
 		int upper_bound = (int)round((sensor_yaw_body_deg  + math::degrees(distance_sensor.h_fov / 2.0f)) / BIN_SIZE);
 
+		const Quatf q_vehicle_attitude(vehicle_attitude);
+		const Quatf q_sensor(Quatf(cosf(sensor_yaw_body_rad / 2.f), 0.f, 0.f, sinf(sensor_yaw_body_rad / 2.f)));
 
-		// rotate vehicle attitude into the sensor body frame
-		Quatf attitude_sensor_frame = vehicle_attitude;
-		attitude_sensor_frame.rotate(Vector3f(0.f, 0.f, sensor_yaw_body_rad));
-		float sensor_dist_scale = cosf(Eulerf(attitude_sensor_frame).theta()); // verify
+		matrix::Vector3f forward_vector(1.0f, 0.0f, 0.0f);
```

Comment:
with
```suggestion
		const Vector3f forward_vector(1.0f, 0.0f, 0.0f);
```

with [this](https://github.com/PX4/PX4-Autopilot/blob/7cdf8eee116b5e5cdaef56b216febd2e360d5748/src/lib/collision_prevention/CollisionPrevention.cpp#L43) on top of the cpp file you don't need to specify that Vector2f and Vector3f are from the matrix namespace.

Also, it's not mandatory but I usually find that adding `const` to all those intermediate variables makes it clearer that they're not modified later.